test/runtests.jl

```diff
@@ -36,7 +36,7 @@ end
     @test Showoff.format_fixed_scientific(0.012345678, 4, true) == "12.346×10⁻³"
     @test Showoff.format_fixed_scientific(0.012345678, 4, false) == "1.2346×10⁻²"
     @test Showoff.format_fixed_scientific(-10.0, 4, false) == "-1.0000×10¹"
-    @test Showoff.format_fixed_scientific(-10.0, 4, false) == "-1.0000×10¹"
+    @test Showoff.format_fixed_scientific(-10.0, 4, true) == "-10.000×10⁰"
```

Comment:
It was same with line 38, thats why I just modified this line in accordance with the bug you found